src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -114,29 +125,253 @@ Template:
+| Magnetization transfer ratio               | MTR     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer ratio map. _Associated output suffixes_: MTRmap                                                                                                                                                                                                                                                                                                                                                                                 |
+| Magnetization transfer saturation          | MTS     | Grouping | Anatomical images for calculating a semi-quantitative magnetization transfer saturation index map. The MTS method involves three sets of anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff) and flip angle ([Helms et al. 2008](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.21732)). _Associated output suffixes_: T1map, MTsat                                                                                                                                                                                               |
+| Multi-parametric mapping                   | MPM     | Grouping | Anatomical images for multiparametric mapping (a.k.a hMRI). The MPM approaches involves the acquisition of highly-similar anatomical images that differ in terms of application of a magnetization transfer RF pulse (MTon or MToff), flip angle and (optionally) echo time and magnitue/phase parts ([Weiskopf et al. 2013](https://www.frontiersin.org/articles/10.3389/fnins.2013.00095/full)). See [here](https://owncloud.gwdg.de/index.php/s/iv2TOQwGy4FGDDZ) for suggested MPM acquisition protocols.,_Associated output suffixes_:R1map, R2starmap, MTsat, PDmap, T1map, T2starmap                                               |
+
+For example:
+
+```Text
+sub-01_fa-1_VFA.nii.gz
+sub-01_fa-1_VFA.json
+sub-01_fa-2_VFA.nii.gz
+sub-01_fa-2_VFA.json
+```
+
+Please see the [entity table appendix]() for the REQUIRED and OPTIONAL entities
+for each `grouping suffix`. 
+
+Note that every image in a **grouped scan collection** has the same `_<suffix>` 
+as they are supposed to be used together. Although the acquisitions will have
+many identical acquisition parameters, only one-to-one-mapping is allowed
+between a `.json`-sidecar file and an image in a grouped scan collection.
+This follows directly from the
+[inheritance principles of BIDS](../02-common-principles.md#the-inheritance-principle):
+parameter values that are identical across a set of grouped scans will still
+have to be stored separately in each `.json`-sidecar file.
+
+##### qMRI map suffixes 
+
+**Function:**
+
+Denotes the parameter contained within an individual file of a
+quantitative parametric image.
+
+Changes to the specification is REQUIRED to
```

Comment:
```suggestion
Changes to the specification are REQUIRED to
```

Reply:
@Gilles86 - if you give this a :+1: vote then I think we can go ahead and commit